app/assets/javascripts/components/SimpleMenuRow.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeAngularComponent } from './utils';
+
+function SimpleMenuRow({
+  action,
+  label,
+  subtitle,
+  desc,
+  circle,
+  circleAlign,
+  faded,
+  disabled,
+  stylekitClass
+}) {
+  return (
+    <div onClick={action} className="sk-menu-panel-row row" title={desc}>
+      <div className="sk-menu-panel-column">
+        <div className="left">
+          {circle && (!circleAlign || circleAlign === 'left') && (
+            <div className="sk-menu-panel-column">
+              <div className={`sk-circle small ${circle}`} />
+            </div>
+          )}
+
+          <div
+            className={`sk-menu-panel-column ${
+              disabled || faded ? 'faded' : ''
+            }`}
+          >
+            <div className={`sk-label ${stylekitClass ? stylekitClass : ''}`}>
+              {label}
+            </div>
+            {subtitle && <div className="sk-sublabel">{subtitle}</div>}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export const ngSimpleMenuRow = makeAngularComponent(
```

Comment:
Perhaps you've looked into this more than I have by this point: is `ng` reserved for official Angular byproducts? Or is this the right way to say that ngSimpleMenuRow is SimpleMenuRow converted to an Angular component?

Reply:
It's not reserved no. I just thought naming them like this was a nice way to communicate their purpose.

Reply:
Perhaps `sn` would be a better prefix. `ng` gives the impression this is coming from Angular core. Hard to find good documentation on this but it's somewhat implicit: https://medium.com/@cironunesdev/angularjs-how-to-name-directives-118ac44b81d4

Reply:
I see your point. However using `sn` would make us lose that implicit documentation that this is an angular component. My first idea was adding an `AngularComponent` suffix, like this: `SimpleMenuRowAngularComponent`. It's kinda long but it's explicit!